templates/shared_services/firewall/porter.yaml

```diff
@@ -0,0 +1,71 @@
+name: tre-shared-service-firewall
+version: 0.0.3
+description: "A Azure TRE Firewall shared service"
+registry: azuretre
+dockerfile: Dockerfile.tmpl
+
+credentials:
+  - name: azure_tenant_id
+    env: ARM_TENANT_ID
+  - name: azure_subscription_id
+    env: ARM_SUBSCRIPTION_ID
+  - name: azure_client_id
+    env: ARM_CLIENT_ID
+  - name: azure_client_secret
+    env: ARM_CLIENT_SECRET
+
+parameters:
+  - name: tre_id
+    type: string
+    description: "The ID of the parent TRE instance e.g., mytre-dev-3142"
+  - name: azure_location
+    type: string
+    description: "Azure location (region) to deploy to"
+  - name: tfstate_resource_group_name
+    type: string
+    description: "Resource group containing the Terraform state storage account"
+  - name: tfstate_storage_account_name
+    type: string
+    description: "The name of the Terraform state storage account"
+  - name: tfstate_container_name
+    type: string
+    default: "tfstate"
+    description: "The name of the Terraform state storage container"
+
+mixins:
+  - exec
+  - terraform:
+      clientVersion: 1.0.4
```

Comment:
Perhaps a good opportunity to also update this to 1.1.5 (or 6 even)?
(same comment for the other bundles as well)

Reply:
1.1.5 as it's the same version as in the DC